extensions/project-descriptor.md

```diff
@@ -7,89 +7,130 @@ A project descriptor is a file that MAY contain configuration for apps, services
 <!-- Using https://github.com/yzhang-gh/vscode-markdown to manage toc -->
 - [Project Descriptor](#project-descriptor)
   - [Table of Contents](#table-of-contents)
-  - [Schema](#schema)
-  - [`[project]`](#project)
-    - [`[[project.licenses]]`](#projectlicenses)
-  - [`[build]`](#build)
-    - [`[build.include]` and `[build.exclude]`](#buildinclude-and-buildexclude)
-    - [`[[build.buildpacks]]`](#buildbuildpacks)
-    - [`[[build.env]]`](#buildenv)
-  - [`[metadata]`](#metadata)
+  - [Schema Version API Version](#schema-version-api-version)
+  - [Special Value Types](#special-value-types)
+  - [Top Level Tables](#top-level-tables)
+    - [Non-`_` Tables](#non-_-tables)
+    - [`_`](#_)
+      - [`_.licenses` (optional)](#_licenses-optional)
+      - [`_.metadata` (optional)](#_metadata-optional)
+    - [`io.buildpacks` (optional)](#iobuildpacks-optional)
+      - [`io.buildpacks.build.include` (optional) and `io.buildpacks.build.exclude` (optional)](#iobuildpacksbuildinclude-optional-and-iobuildpacksbuildexclude-optional)
+      - [`io.buildpacks.build.buildpacks` (optional)](#iobuildpacksbuildbuildpacks-optional)
+      - [`io.buildpacks.build.env` (optional)](#iobuildpacksbuildenv-optional)
   - [Example](#example)
 
-## Schema
+## Schema Version API Version
 
-The TOML schema of the project descriptor is the following:
+This document specifies Project Descriptor Schema API Version `0.2`.
```

Comment:
Do we just want to call this a schema version everywhere instead of a schema API version?